doc/starting.rst

```diff
@@ -76,6 +76,10 @@ enabling you to connect via ``telnet``, use the following code::
     from pudb.remote import set_trace
     set_trace(term_size=(80, 24))
 
+The terminal size can be defined via the environment variable as well::
+
+    export PUDB_TERM_SIZE=80x24
+
```

Comment:
Please document which takes precedence in the various cases.

Reply:
Done!

In your opinion, does it make sense to support something like `PUDB_RDB_REVERSE` for the sake of completeness?

Reply:
What would the use case be? I.e. in what situation would you be unable to initiate "reverse" debugging in the existing ways?

Reply:
The main use-case is using the generic `breakpoint()` to invoke the debugger. Then the debugger parameters have to be defined as env vars (`PYTHONBREAKPOINT`, etc)

Reply:
Not sure I'm sold. If at all, let's do that in a separate PR.